Changed all controllers to REST controllers

AmanEfer opened the pull request:
All controllers were changed to REST controllers.

AmanEfer committed:
All controllers were changed to REST controllers

vicria commented on src/main/java/ru/kata/spring/security/demo/controllers/AdminController.java:
```diff
@@ -3,24 +3,23 @@
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.*;
```
старайся не ставить звездочку
в Идеи надо In the Class count to use import with '*' and Names count to use static import with '*' выставить 9999 https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports

AmanEfer replied:
Готово

vicria commented on src/main/java/ru/kata/spring/security/demo/services/UserServiceImpl.java:
```diff
@@ -46,15 +46,18 @@ public UserDto getUserByUsername(String username) {
     @Override
     @Transactional
-    public void updateUser(Long id, User user) {
+    public UserDto updateUser(Long id, User user) {
```
а зачем отдельный Long id, если id вроде есть в User user?

AmanEfer replied:
Исправил

vicria reviewed (approved):
всё ок. небольшие вопросы только

AmanEfer committed:
- rename packages
- create modules 'crud' and 'telegram'
- create poms into modules and change root pom

AmanEfer committed:
Added some features into telegram module:
- made some changes in all poms cause they didn't work
- added functionality for telegram
- added RestToCrud to connect with CRUD

AmanEfer committed:
Update application.properties
